File: server/php/src/Application/Domain/Payments/PaymentDetails/FraudResults.php
<?php

namespace OnlinePayments\ExampleApp\Application\Domain\Payments\PaymentDetails;

final class FraudResults
{
    public ?string $fraudServiceResult = null;

    public function toArray(): array
    {
        return [
            'fraudServiceResult' => $this->fraudServiceResult,
        ];
    }
}

File: server/ruby/app/business/domain/payments/payment_details/fraud_results.rb
module Business
  module Domain
    module Payments
      module PaymentDetails
        class FraudResults
          attr_accessor :fraud_service_result

          def initialize(fraud_service_result: nil)
            @fraud_service_result = fraud_service_result
          end

          def to_h
            {
              'fraudServiceResult' => @fraud_service_result
            }
          end
        end
      end
    end
  end
end